src/Base/IntVect.cpp

```diff
@@ -28,8 +28,10 @@ namespace
         py::class_< iv_type > py_iv(m, iv_name.c_str());
         py_iv
             .def("__repr__",
-                 [iv_name](const iv_type&) {
-                     return "<amrex." + iv_name + ">";
+                 [iv_name](const iv_type& iv) {
+                     std::stringstream s;
+                     s << iv;
+                     return "<amrex." + iv_name + s.str() + ">";
```

Comment:
Would this be more natural?
I forgot what the `<<` for `iv` implements
```suggestion
                     return "<amrex." + iv_name + ", " + s.str() + ">";
```

Reply:
Either way is fine. As I have it, it would print something like
`<amrex.IntVect(10,20,30)>`
versus
`<amrex.IntVect, (10,20,30)>`

Reply:
Thanks, got it.